src/middlewared/middlewared/etc_files/subgid.mako

```diff
@@ -0,0 +1,4 @@
+<%
+from middlewared.api.base.types.user import INCUS_IDMAP_MIN, INCUS_IDMAP_COUNT
+%>\
+0:${INCUS_IDMAP_MIN}:${INCUS_IDMAP_COUNT}
```

Comment:
Docker uses these sames files so I'm curious how this will affect it

Reply:
I can test. I am guessing docker uses "newuidmap" tool to manipulate that file.
Is running any app enough or something else required?

Reply:
Actually I am thinking docker will only set the subuid/subgid for a namespace they create, I will test with several privileged apps.

Reply:
Did some testing, it seems we are not using user namespaces in our docker/Apps implementation at all.
I did run a privileged custom ix-app and had no apparent issues.

https://docs.docker.com/engine/security/userns-remap/

Using user namespaces with docker will require explicit interaction with that file which we do not.